scripts/get_touchpad_info.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# A script to dump info about the touchpad
+
+uid=$(id -u)
+if [ $uid -ne 0 ]; then
+	echo "This script must be run as root!"
+	exit
+fi
+
+echo "Kernel log:"
+dmesg
+
+echo "Installing dependencies"
+apt-get update
+apt-get install -y i2c-tools acpi-call-dkms 
+
+echo "running i2cdetect (quick write mode)"
+i2cdetect -y -q 1
+
+echo "running i2cdetect (receive byte mode)"
+i2cdetect -y -r 1
+
+echo "Loading module acpi_call"
+modprobe acpi_call
+
+devpath=$(dmesg | grep hid-generic | awk '{print $4}' | cut -d ':' -f 1-3)
```

Comment:
This also won't work as expected. dmesg from my PC:

```
[    1.512777] hid-generic 0003:413C:2113.0001: input,hidraw0: USB HID v1.11 Keyboard [Dell KB216 Wired Keyboard] on usb-0000:00:1a.0-1.1/input0
[    1.572904] hid-generic 0003:413C:2113.0002: input,hidraw1: USB HID v1.11 Device [Dell KB216 Wired Keyboard] on usb-0000:00:1a.0-1.1/input1
[    1.631587] hid-generic 0003:413C:301A.0003: input,hidraw2: USB HID v1.11 Mouse [PixArt Dell MS116 USB Optical Mouse] on usb-0000:00:1a.0-1.2/input0
```

We surely are not interested in USB HID devices...